arithmetization/src/main/java/net/consensys/linea/zktracer/module/blockdata/Blockdata.java

```diff
@@ -70,7 +70,7 @@ public void traceStartConflation(final long blockCount) {
             + 1
             + 6 // for NUMBER
             + 1 // for DIFFICULTY
-            + Bytes.minimalBytes(GAS_LIMIT_MAXIMUM).size() * 4 // for GASLIMIT
+            + (bigIntegerToBytes(GAS_LIMIT_MAXIMUM).size() * 4) // for GASLIMIT
```

Comment:
@letypequividelespoubelles This is because `GAS_LIMIT_MAXIMUM` is now forced to be a `BigInteger` no matter what chain is used.